### PR TITLE
Improved the model by making ops log process local variable

### PR DESCRIPTION
**Pull request by muratdem**
also fixed the empty writeset issue

### CODE REVIEW

File: specifications/KeyValueStore/KVsnap.tla [hunk 1]
```diff
@@ -12,9 +12,12 @@ CONSTANTS   Key,            \* The set of all keys.
 \* Instantiating ClientCentric enables us to check transaction isolation guarantees this model satisfies
 \* https://muratbuffalo.blogspot.com/2022/07/automated-validation-of-state-based.html         
 CC == INSTANCE ClientCentric WITH Keys <- Key, Values <- TxId \union {NoVal}          
-
+    
+\* for instantiating the ClientCentric module
 wOp(k,v) == CC!w(k,v)
 rOp(k,v) == CC!r(k,v)
+InitialState == [k \in Key |-> NoVal]   
+SetToSeq(S) == CHOOSE f \in [1..Cardinality(S) -> S] : IsInjective(f)
```
Comment (lemmy) [inline specifications/KeyValueStore/KVsnap.tla:20]: Consider using https://github.com/tlaplus/CommunityModules/blob/f070b68b02cf5b6d085797402374d85f739c167e/modules/SequencesExt.tla#L22-L27

Comment (muratdem) [reply specifications/KeyValueStore/KVsnap.tla:20]: Will keep this in mind for future models. I don't want to change manifest.json and depend on SequencesExt now.

Comment (lemmy) [reply specifications/KeyValueStore/KVsnap.tla:20]: Automation gets in the way. :-(

**Comment by muenchnerkindl** [comment]
Sure, I'll merge this version.

**Pull request merged**